src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -43,6 +54,13 @@
     "tx4power":     43,
 }
 
+PSU_SENSOR_PART_ID_MAP = {
```

Comment:
Is this common for all vendors?

Reply:
Yes, all sensor values can get from platform API. If vendor doesn't implement the API, we will give it a None value in psud and it won't be displayed in mib.